### PR TITLE
Content migration batch 11

### PR DESCRIPTION
**Pull request by mattletford**
Migrating content

https://developers.google.com/web/fundamentals/performance/get-started
https://web.dev/performance-get-started

https://developers.google.com/web/fundamentals/performance/get-started/graphicalcontent-4
https://web.dev/performance/get-started-graphicalcontent-4

https://developers.google.com/web/fundamentals/performance/get-started/httpcaching-6
https://web.dev/performance/get-started-httpcaching-6

https://developers.google.com/web/fundamentals/performance/get-started/httprequests-5
https://web.dev/performance/get-started-httprequests-5

https://developers.google.com/web/fundamentals/performance/get-started/textcontent-3
https://web.dev/performance-get-started-textcontent-3

@rachelandrew

### CODE REVIEW

File: src/site/content/en/blog/performance-get-started-httpcaching-6/index.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+---
+
+When someone visits a website, everything the site needs to display and operate has to come
+from somewhere. All the text, images, CSS styles, scripts, media files, and so on must be
+retrieved by the browser for display or execution. You can give the browser choices about
+where it can retrieve a resource from, and that can make a big difference in your page's
+load speed.
+
+The first time a browser loads a web page, it stores the page resources in the
+[HTTP Cache](/web/fundamentals/performance/optimizing-content-efficiency/http-caching).
```
Comment (rachelandrew) [inline src/site/content/en/blog/performance-get-started-httpcaching-6/index.md:16]:
```suggestion
[HTTP Cache](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/http-caching).
```

**Review by rachelandrew — CHANGES_REQUESTED** [review]
There's a UR to fix in this.

**Review by rachelandrew — CHANGES_REQUESTED** [review]
An issue has come up when I ran tests, see the comment inline.

File: src/site/content/en/blog/performance-get-started-textcontent-3/index.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+already been individually compressed generally provides no significant size reduction.
+
+Unlike desktop or browser-based local compression, Gzip works at the server to identify and
+process certain file types that you specify. While all modern browsers support Gzip compression
+for HTTP requests, you must properly configure your server to deliver the compressed resource
+when it is requested. Different server types, of course, have different setup requirements.
+You configure the Apache server, for example, via the `.htaccess` file, which would include
+something like this.
+
+```apacheapacheconf
```
[the remark this answers is not in the document]
Comment (mattletford) [reply src/site/content/en/blog/performance-get-started-textcontent-3/index.md]: @rachelandrew changed language name

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**